app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -638,7 +642,12 @@ def new_interface(attr = { })
     end
 
     def new_volume(attr = { })
-      client.volumes.new attr.merge(:size_gb => 10)
+      {
+        :thin => true,
```

Comment:
Without looking, don't we have some option to control whether it's thin or thick?

You're also completely ignoring the `attr` now. Is that intended as well?

Reply:
No, thin is just the default. We can change it as per convenience.

Reply:
Ah I didn't notice attr being unused. Thanks!!

Reply:
This was due to mismatch of Hash/Object type when setting controller_key for volumes, but since I handled that [here](https://github.com/fog/fog-vsphere/pull/300/commits/26303a459db47ef9133bcc31dcc1be1465033014#diff-49106de6ade06895f1a509db7c9cd040ed76bfdb472548d8b99efc0468dce548R363), I have switched it back to thr way it was before